redis.go

```diff
@@ -35,6 +35,18 @@ func (c *baseClient) conn() (*pool.Conn, error) {
 			_ = c.connPool.Remove(cn, err)
 			return nil, err
 		}
+		// If readonly then send readonly command for the connection
+		if c.opt.ReadOnly {
+			cmd := newKeylessStatusCmd("READONLY", isReadOnly)
```

Comment:
`READONLY` is only for the connection it was sent on so every new connection won't allow readonly.
Not sure it's the best place for that though but I can't find a better one.